### ISSUE TITLE
update documentation on database commands

### ISSUE THREAD

**Issue by fdncred** [nushell/nushell#5875]
### Related problem

after https://github.com/nushell/nushell/pull/5838, documentation is out of date.
also, probably need a specific section on sqlite db commands and examples.

### Describe the solution you'd like

updated docs

**Comment by elferherrera** [comment]
We don't have a page in the book. Do we?

**Comment by fdncred** [comment]
> We don't have a page in the book. Do we?

I was looking for something in the advanced section next to dataframes and I didn't see anything. Then I searched and just got the command help. So, I guess we don't.

**Comment by elferherrera** [comment]
Nope. I haven't written one. I want to finish the insert commands before writing it
